frontend/dockerfile/docs/reference.md

```diff
@@ -1804,32 +1808,50 @@ COPY --from=nginx:latest /etc/nginx/nginx.conf /nginx.conf
 The source path of `COPY --from` is always resolved from filesystem root of the
 image or stage that you specify.
 
-### COPY --chown --chmod
+### COPY --chmod
 
-> [!NOTE]
-> Only octal notation is currently supported. Non-octal support is tracked in
-> [moby/buildkit#1951](https://github.com/moby/buildkit/issues/1951).
+```dockerfile
+COPY [--chmod=<perms>] <src> ... <dest>
+```
+
+The `--chmod` flag supports octal notation (e.g., `755`, `644`).
+Symbolic notation (e.g., `+x`, `g=u`) requires the `docker/dockerfile:1-labs`
+syntax directive. This is useful when octal isn't flexible enough. For example,
+`u=rwX,go=rX` sets directories to 755 and files to 644, while preserving the
```

Comment:
Maybe link to `man chmod` like https://man.freebsd.org/cgi/man.cgi?chmod for a longer explanation.